examples/eos-transfers/index.js

```diff
@@ -23,4 +23,11 @@ const actionWatcher = new BaseActionWatcher(
   500,
 )
 
+// If using MongoActionReader you need to call
```

Comment:
We should make nextBlock in the mongo action reader throw an error if it hasn't been initialized as well.  That way we can provide the user context explicit error messaging in the event they fail to read documentation.

E.g.
 - initialize is an interanal boolean initially set to false
 - after calling initialize initilize is set to true
 - if the user trys to use "nextBlock" we do a check in that method to see if initialize is set to true. If it isn't then we should throw an error "MongoActionReader" must be initialized before fetching blocks.